libs/layer-composer/src/config.ts

```diff
@@ -2,7 +2,7 @@ export const API_GATEWAY_VERSION =
   process.env.API_GATEWAY_VERSION ||
   process.env.REACT_APP_API_GATEWAY_VERSION ||
   process.env.NEXT_PUBLIC_API_GATEWAY_VERSION ||
-  'v1'
+  'v2'
```

Comment:
Do you think it would be worth passing API version in the `GlobalGeneratorConfig` instead?

Reply:
not sure because the tilesUrl should be resolved by [resolve-dataviews-generators](https://github.com/GlobalFishingWatch/frontend/blob/fishing-map/v2-api-migration/libs/dataviews-client/src/resolve-dataviews-generators.ts#L240), I'll merge and review later if this is still used